front/app/helpers/is-browser-outdated.js

```diff
@@ -0,0 +1,6 @@
+export default Ember.Helper.helper(function(){
+    if(!window.fetch){
+        return true;
+    }
+    return false;
+});
```

Comment:
Maybe we can make use of something like https://browser-update.org/ ? I see an npm package: https://www.npmjs.com/package/ember-cli-browser-update but I don't know if it works with the version of Ember we're on.